client.lua

```diff
@@ -1300,7 +1300,7 @@ RegisterNetEvent('ox_inventory:setPlayerInventory', function(currentDrops, inven
 			price = data.price,
 			invId = id,
 			nearby = nearbyLicense,
-			message = ('**%s**  \n%s'):format(locale('purchase_license', data.name), locale('interact_prompt', GetControlInstructionalButton(0, 38, true):sub(3)))
+			message = ('**%s**  \n%s'):format(locale('purchase_license', data.label), locale('interact_prompt', GetControlInstructionalButton(0, 38, true):sub(3)))
```

Comment:
... and if there _is_ no label assigned (as default)? Your label might also be "Weapon License", making the prompt "Weapon License license".

Reply:
Oh, I didn't realize that. I've been editing this for a long time with every update. I forgot that I just added a label to my license table.